### PR TITLE
Fixing security vulnerability 

### PR DESCRIPTION
**Pull request by karen-avetisyan-mc**
#### Description
Upgrading Cryptography to use 42.0.0.0 or above.
Replacing deprecated load_pkcs12

### CODE REVIEW

**Comment by ShimonaR-MC** [comment]
Cryptography 42.0.0 requires Python 7+. Won't work with python 6.

**Comment by danny-gallagher** [comment]
> Cryptography 42.0.0 requires Python 7+. Won't work with python 6.

We will also need to update the README with the new version of python supported, currently it says 3.6+

**Comment by karen-avetisyan-mc** [comment]
Readme updated and builds are passing

**Comment by karen-avetisyan-mc** [comment]
> Cryptography 42.0.0 requires Python 7+. Won't work with python 6.
We will start from 3.8

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -15,16 +15,14 @@ jobs:
...
           - 3.9
         include:
-            - os: "ubuntu-latest"
-            # There is no build for Python 3.6 in ubuntu>20.04.
-            # This unclude can be removed when we drop support for Python 3.6.
-            - os: "ubuntu-20.04"
-              python-version: "3.6"
+          - os: "ubuntu-latest"
+          # There is no build for Python 3.9 in ubuntu>20.04.
+          # This unclude can be removed when we drop support for Python 3.9.
```
Comment (danny-gallagher) [inline .github/workflows/test.yml]: Is this comment still relevant?

Comment (karen-avetisyan-mc) [reply .github/workflows/test.yml]: changed it to 3.9,

**Review by danny-gallagher — APPROVED** [review]
LGTM!

**Pull request merged**